### DISCUSSION TITLE
ReactiveRecyclerViewViewHolder was deprecated in API level 23 ?

### DISCUSSION THREAD

**Discussion by Lapinou42** [reactiveui/ReactiveUI#3672] [Q&A] [answered]
Hello, 

We got some warnings in our pipeline that comes from: https://github.com/reactiveui/ReactiveUI/blob/8f11df403a962cb8a2a9539c8134084c65db259e/src/ReactiveUI.AndroidX/ReactiveRecyclerViewViewHolder.cs#L40-L43

I didn't find any alternative to this. 
The strange thing is there is pretty much the same code in this class: https://github.com/reactiveui/ReactiveUI/blob/8f11df403a962cb8a2a9539c8134084c65db259e/src/ReactiveUI.AndroidSupport/ReactiveRecyclerViewViewHolder.cs#L37-L39

Is it normal that the deprecated method is in the AndroidX environment instead of AndroidSupport ?

Thank you for your feedback.

**Comment by ChrisPulman** [comment, marked as answer]
This will be resolved in the next release, there is an obsolete property in use currently which is now updated to a alternative recommended property.

**Reply by Lapinou42** [reply]
@ChrisPulman, 

Great, many thanks to you :)
